src/librustdoc/json/conversions.rs

```diff
@@ -6,7 +6,7 @@
 
 use rustc_abi::ExternAbi;
 use rustc_ast::ast;
-use rustc_attr_parsing::DeprecatedSince;
+use rustc_attr_data_structures::{self as attrs, DeprecatedSince};
```

Comment:
Any reason for the rename here?

Reply:
If I change https://github.com/rust-lang/rust/blob/9a7e19f2b66c472811b4bff02695c7a03f3f4151/src/librustdoc/json/conversions.rs#L156

to 

```rust
pub(crate) fn from_deprecation(deprecation: rustc_attr_data_structures::Deprecation) -> Deprecation {
```

Then rustfmt will turn that into

```rust
pub(crate) fn from_deprecation(
    deprecation: rustc_attr_data_structures::Deprecation,
) -> Deprecation {
}
```

And I can't import `rustc_attr_data_structures::Deprecation` because that conflicts with rustdoc's `Deprecation`.

IMO `as attrs` makes for the nicest diff.